docs/markdown/Reference-manual.md

```diff
@@ -536,6 +538,10 @@ Project supports the following keyword arguments.
 
 Runs the command specified in positional arguments. Returns [an opaque object](#run-result-object) containing the result of the invocation. The script is run from an *unspecified* directory, and Meson will set three environment variables `MESON_SOURCE_ROOT`, `MESON_BUILD_ROOT` and `MESON_SUBDIR` that specify the source directory, build directory and subdirectory the target was defined in, respectively.
 
+The keyword arguments are as follows:
+
+- `in_builddir` If set to `true`, run the command in the build directory.
```

Comment:
This was probably not documented by design (as there was no test either). In the above paragraph it says that the command is run from an unspecified directory. These two don't really work together. We probably need a solution that solves both of these issues or neither.